tools/intellij-plugin/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+  id 'org.jetbrains.intellij' version '1.16.0'
+  id 'org.jetbrains.kotlin.jvm' version "1.9.10"
+}
+
+group = 'org.dxos'
+version = '0.0.1'
+
+repositories {
+  mavenCentral()
+  jcenter()
+}
+
+sourceCompatibility = JavaVersion.VERSION_17
+targetCompatibility = JavaVersion.VERSION_17
+
+configure([compileKotlin, compileTestKotlin]) {
+  kotlinOptions {
+    jvmTarget = "17"
+  }
+}
+
+runIde {
+  systemProperty("idea.platform.prefix", "PhpStorm")
+  minHeapSize = "512m"
+  maxHeapSize = "3g"
+}
+
+// See https://github.com/JetBrains/gradle-intellij-plugin/
```

Comment:
this will work for webstorm?

Reply:
yeah, should run with any intellij IDE where NodeJS and JS plugins are installed and those should be bundled with webstorm